src/style.css

```diff
@@ -0,0 +1,24 @@
+* {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
```

Comment:
You don't need margin right for the last element, it's useless, get habit of removing unnecessary properties